c/cert/test/rules/ERR33-C/test.c

```diff
@@ -0,0 +1,137 @@
+#include <inttypes.h>
```

Comment:
was just kind of skimming results in openpilot, saw this line `float scale = getenv("SCALE") != NULL ? std::stof(getenv("SCALE")) : 1.0;` come up in qt_window.cc:15:53, is this a FP?